lib/ret/hub.ex

```diff
@@ -123,10 +125,21 @@ defmodule Ret.Hub do
 
   def changeset_for_new_host(%Hub{} = hub, host), do: hub |> cast(%{host: host}, [:host])
 
+  def changeset_for_creator_assignment(
+        %Ret.Hub{creator_assignment_token: expected_token} = hub,
+        account,
+        token
+      )
+      when expected_token != nil and expected_token == token do
+    hub |> cast(%{}, []) |> add_account_to_changeset(account)
```

Comment:
I've seen you use `cast` like this a few times, not sure if you're intentionally doing it but if you have no params to cast, you could just use [`Ecto.Changeset.change/2`](https://hexdocs.pm/ecto/2.2.11/Ecto.Changeset.html#change/2). Same in the overload below.

Reply:
ah good to know